distribution/docs/src/main/resources/_contents/_security-framework/security-rest-intro-contents.adoc

```diff
@@ -0,0 +1,1255 @@
+                </RequestedSecurityToken>
+                <RequestedAttachedReference>
+                    <ns3:SecurityTokenReference xmlns:wsse11="http://docs.oasis-open.org/wss/oasis-wss-wssecurity-secext-1.1.xsd" wsse11:TokenType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0">
+                        <ns3:KeyIdentifier ValueType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLID">_BDC44EB8593F47D1B213672605113671</ns3:KeyIdentifier>
+                    </ns3:SecurityTokenReference>
+                </RequestedAttachedReference>
+                <RequestedUnattachedReference>
+                    <ns3:SecurityTokenReference xmlns:wsse11="http://docs.oasis-open.org/wss/oasis-wss-wssecurity-secext-1.1.xsd" wsse11:TokenType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0">
+                        <ns3:KeyIdentifier ValueType="http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLID">_BDC44EB8593F47D1B213672605113671</ns3:KeyIdentifier>
+                    </ns3:SecurityTokenReference>
+                </RequestedUnattachedReference>
+                <wsp:AppliesTo xmlns:wsp="http://schemas.xmlsoap.org/ws/2004/09/policy" xmlns:wst="http://docs.oasis-open.org/ws-sx/ws-trust/200512">
+                    <wsa:EndpointReference xmlns:wsa="http://www.w3.org/2005/08/addressing">
+                        <wsa:Address>https://server:8993/services/SecurityTokenService</wsa:Address>
+                    </wsa:EndpointReference>
+                </wsp:AppliesTo>
+                <Lifetime>
+                    <ns2:Created>2013-04-29T18:35:11.444Z</ns2:Created>
+                    <ns2:Expires>2013-04-29T19:05:11.444Z</ns2:Expires>
+                </Lifetime>
+            </RequestSecurityTokenResponse>
+        </RequestSecurityTokenResponseCollection>
+    </soap:Body>
+</soap:Envelope>
+----
+
+===== UsernameToken Bearer SAML Security Token Request/Response
+
+To obtain a SAML assertion to use in secure communication to ${branding}, a RequestSecurityToken (RST) request has to be made to the STS.
+
+A Bearer SAML assertion is automatically trusted by the endpoint.
+The client doesn't have to prove it can own that SAML assertion.
+It is the simplest way to request a SAML assertion, but many endpoints won't accept a KeyType of Bearer.
+
+====== Request
+
+*Explanation*
+
+* WS-Addressing header with Action, To, and Message ID
+* Valid, non-expired timestamp
+* Username Token containing a username and password that the STS will authenticate
+* Issued over HTTPS
+* KeyType of http://docs.oasis-open.org/ws-sx/ws-trust/200512/Bearer
+* Claims (optional): Some endpoints may require that the SAML assertion include attributes of the user, such as an authenticated user's role, name identifier, email address, etc. If the SAML assertion needs those attributes, the `RequestSecurityToken` must specify which ones to include.
+
+.Sample Request
```

Comment:
This sample request and the one further down the page are both using the old format which we don't support anymore. We need to update these.